python/pdo/common/keys.py

```diff
@@ -60,6 +57,7 @@ class TransactionKeys(object) :
     """
```

Comment:
Do we need TransactionKeys after Sawtooth removal? (the comment says "    Wrapper for managing Sawtooth transaction keys").

Also, please see ContractMessage __init__. Channel_id is no longer TransactionKeys. The derivation of channel_id depends on the ledger type, for CCF, it's just nonce (AR: change the comment on line 30 of message.py)

Reply:
> Do we need TransactionKeys after Sawtooth removal? (the comment says " Wrapper for managing Sawtooth transaction keys").

It does not seem so, I have deprecated it. Upon removal, the dependency on secp256k1 can be discarded as well.

Reply:
> Also, please see ContractMessage init. Channel_id is no longer TransactionKeys. The derivation of channel_id depends on the ledger type, for CCF, it's just nonce (AR: change the comment on line 30 of message.py)

Committed https://github.com/hyperledger-labs/private-data-objects/commit/039e4132ae2dedf6d284692fc3dcd190376c3bd1